lib/licensed/sources/gradle.rb

```diff
@@ -122,61 +78,132 @@ def configurations
         end
       end
 
-      def self.add_gradle_license_report_plugins_block(gradle_build_file)
+      # Prefixes the gradle command with the project name for multi-build projects.
+      def format_command(command)
+        if config.source_path != config.root
```

Comment:
I'm not sure that `config.root` is the right option here either.  the [config object's root](https://github.com/github/licensed/blob/master/docs/configuration/configuration_root.md) is agnostic to the requirements or concerns of any particular sources and will be one of (in order of preference):
- the configuration file's directory, when configured as `true`
- a relative path from the configuration file, when configured as a path
- the repository root, if run from a git repository
- the directory that licensed is run from, when no other values are available

Can the gradle CLI report the project name to use here based on the current directory, or at least tell whether the current project is the root or not?

Reply:
Great suggestion! I found a way to find the current project with the cli.